src/ucp/core/ucp_mm.c

```diff
@@ -599,11 +599,33 @@ ucp_memh_rcache_get(ucs_rcache_t *rcache, void *address, size_t length,
     return UCS_OK;
 }
 
+static ucp_md_index_t ucp_mem_get_md_index(ucp_context_h context,
+                                           const uct_md_h md,
+                                           uct_alloc_method_t method)
+{
+    ucp_md_index_t md_index = UCP_NULL_RESOURCE;
+    ucp_md_index_t index;
```

Comment:
why need also "index" variable?
IMO we can just use "md_index" as the loop variable

Reply:
removed index